src/main/scala/uk/gov/hmrc/play/microservice/bootstrap/MicroserviceGlobal.scala

```diff
@@ -35,15 +34,15 @@ trait MicroserviceFilters {
 
   def metricsFilter: MetricsFilter = MetricsFilter
   
-  def authFilter: MicroserviceAuthorisationFilter
+  def authFilter: Option[EssentialFilter]
 
   protected lazy val defaultMicroserviceFilters: Seq[EssentialFilter] = Seq(
-    metricsFilter,
-    microserviceAuditFilter,    
-    loggingFilter,
+    Some(metricsFilter),
```

Comment:
Why are some of these filters Options? It looks like the list is just flattened straight away